datasets/dataset_setup.py

```diff
@@ -436,19 +436,17 @@ def download_imagenet_v2(data_dir):
 
 
 def download_librispeech(dataset_dir, tmp_dir, train_tokenizer):
+  # pylint: disable=line-too-long
   # After extraction the result is a folder named Librispeech containing audio
   # files in .flac format along with transcripts containing name of audio file
   # and corresponding transcription.
-  tmp_librispeech_dir = os.path.join(tmp_dir, 'librispeech')
+  tmp_librispeech_dir = os.path.join(tmp_dir, 'LibriSpeech')
```

Comment:
nit: I'd prefer to keep all filepaths lowercase (to be consistent with the rest of the datasets) unless we have a good reason otherwise

Reply:
As the comment above says, the extracted audio files is inside Librispeech and this is the data_dir that the tokenizer expects.